src/index.css

```diff
@@ -69,38 +56,559 @@ a {
 }
 
 .avatar {
-  max-width: 2.5rem;
-  height: auto;
-  background-color: #dfe1e6;
+  max-height: 3rem;
 }
 
-.bookDetails {
-  margin: 2rem 0 2rem 0;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+.controlBox button {
+  background-color: var(--white-two);
+  font-family: 'Roboto Slab', serif;
+  border: none;
+  outline: none;
+  font-weight: 300;
+  color: #4386bf;
 }
 
-.secondCol {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  gap: 1rem;
+.divider {
+  width: 0.125rem;
+  height: 1.2rem;
+  margin: 0 5% 0 5%;
+  border: solid 1px var(--white-three);
+}
+
+.hrDivider {
+  width: 100%;
+  height: 0.125rem;
+  margin: 2.5rem 0 2.5rem 0;
+  border: solid 1px var(--white-three);
 }
 
-.progressBarAndUpdateButton {
+.bookForm {
   display: flex;
   flex-direction: row;
+  margin-bottom: 3rem;
   justify-content: space-between;
+  width: 100%;
+  padding-top: 2rem;
+  padding-bottom: 4rem;
 }
 
-.progressImg {
-  max-width: 5rem;
-  max-height: 5rem;
+input[type='text'],
+select {
+  border: solid 1px var(--white-three);
+  background-color: var(--white-two);
+  padding-left: 1rem;
+  outline: none;
 }
 
-.bookForm {
+.categoryPage {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem 0 3rem 0;
+}
+
+.addBook {
+  min-width: 10vw;
+  height: 2.5rem;
+  border-radius: 4px;
+  font-family: 'Roboto Slab', serif;
+  background-color: var(--primary-color);
+  outline: none;
+  border: solid 1px var(--white-three);
+  color: white;
+}
+
+/* mobile version */
+@media (width >= 765px) {
+  .navItems {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 5%;
+  }
```

Comment:
- [ ] Good job Styling your bookstore 👏. As for the page , you should make sure your design matches the design 👍. Your design does not fill all the size of the screen as in the Zeplin design , Kindly fix that 😄 

### Your design

![Capture](https://github.com/Rafiduzzaman/bookstore/assets/80458199/6682f1a1-c5ac-4741-ac45-dbc580cde093)

### Zeplin

![Capture0](https://github.com/Rafiduzzaman/bookstore/assets/80458199/c9577a2a-0e14-4489-a2f1-a8d2b87646aa)